backend/open_webui/storage/provider.py

```diff
@@ -145,41 +145,41 @@ def delete_all_files(self) -> None:
 
 class GCSStorageProvider(StorageProvider):
     def __init__(self):
```

Comment:
I am not understanding the logic behind this client instantiation. Maybe a few question to try to understand, if the `GOOGLE_APPLICATION_CREDENTIALS_JSON` is given does the user still needs to provide a `GCS_PROJECT_ID`? 
I would put the logics such that, if no `GOOGLE_APPLICATION_CREDENTIALS_JSON` are given and all the other necessary ones are, then get the credentials from the VM.